src/pages/AddMembersPage/AddMembersPage.tsx

```diff
@@ -88,12 +95,28 @@ const AddMembersContents = ({
 
   const [preloadedMembers, setPreloadedMembers] = useState<NewMember[]>([]);
 
+  const [guildInfo, setGuildInfo] = useState<GuildInfo | undefined>(undefined);
```

Comment:
Can we add a link to the docs @CryptoGraffe

Reply:
i'm going to defer this for now but will add in a follow-on